projects/demo/src/modules/components/editor/highlight/editor-highlight.module.ts

```diff
@@ -0,0 +1,35 @@
+import {CommonModule} from '@angular/common';
+import {NgModule} from '@angular/core';
+import {FormsModule, ReactiveFormsModule} from '@angular/forms';
+import {RouterModule} from '@angular/router';
+import {TuiAddonDocModule, tuiGenerateRoutes} from '@taiga-ui/addon-doc';
+import {TuiEditorModule, TuiEditorSocketModule} from '@taiga-ui/addon-editor';
+import {TuiActiveZoneModule} from '@taiga-ui/cdk';
+import {
+    TuiButtonModule,
+    TuiHostedDropdownModule,
+    TuiLinkModule,
+    TuiSvgModule,
+} from '@taiga-ui/core';
+
+import {ExampleTuiEditorHighlightComponent} from './editor-highlight.component';
+import {TuiEditorHighlightExample1} from './examples/1';
+
+@NgModule({
+    imports: [
+        CommonModule,
+        TuiActiveZoneModule,
+        TuiSvgModule,
+        TuiLinkModule,
+        TuiHostedDropdownModule,
```

Comment:
Does this example actually need all these imports ?

Reply:
fixed